sites/footers/footers-en.md

```diff
@@ -0,0 +1,84 @@
+---
+altLangPage: footers-fr.html
+contextualFooter:
+  title: "[Contextual footer header]"
+  links:
+  - url: http://canada.ca/en
+    text: Contextual link 1
+  - url: http://canada.ca/en
+    text: Contextual link 2
+  - url: http://canada.ca/en
+    text: Contextual link 3
+dateModified: 2023-01-16
+includes:
+  footer: edge
+language: en
+secondlevel: false
+share: true
+title: Footer
+---
+<div class="wb-prettify all-pre hide"></div>
+
+{% include alert-softlaunch.html component="site footer" version="4" %}
+
+The global (site-wide) footer at the bottom of each web page contains up to 3 bands:
+* Contextual band
+* Main band
+* Sub-footer band
+
+Consult the Canada.ca Design System for guidance on which footer elements or custom implementations to use based on the type of page you're creating:
+* [Global footer - Canada.ca Design System](https://design.canada.ca/common-design-patterns/site-footer.html)
+
+## Latest changes
+
+### Version 4.0
+
+The following changes have been made since version 2.0 and 3.0:
+* Added a contextual band above the main band.
+* The `.landscape` class was renamed to `.gc-main-footer`
+* Added a new heading "Government of Canada" in the main band.
+* Added row of government-wide links.
+* Added short white horizontal rule to separate government-wide links and theme and topics links.
+* Modified the list of themes and topics links.
+* The classes for the themes and topics links list were updated to `list-col-xs-1 list-col-sm-2 list-col-md-3`.
+    * Some lists now use the CSS "Flexbox" feature which distributes links horizontally instead of vertically.
+    * Links are now displayed in one (1) column on extra small screens, in two (2) columns on small screens, and in three (3) columns on medium screens and up.
+* The `.brand` class was renamed to `.gc-sub-footer`
+* The links "Terms and conditions" and "Privacy" have to remain visible at all times.
+* Removed "Top of page" anchor.
+* Complete rework of the footer's headings for accessibility.
+
+## Reference implementation
+
+### Expected output code - Complete footer
+
+{%- include variable-core.liquid -%}
+{% highlight html %}
+	{%- include footers/footer.html -%}
+{% endhighlight %}
+
+## Alternate implementations
+
+### Standard pages
+
+* [Main band and sub-footer band only](https://wet-boew.github.io/GCWeb/sites/footers/no-footer-contextual-en.html)
```

Comment:
Here a suggestion on how we can include this page in the list of working example

```suggestion
* [Contextual band, main band and sub-footer band](#reference-implementation) - As demoed in this page
* [Main band and sub-footer band only](https://wet-boew.github.io/GCWeb/sites/footers/no-footer-contextual-en.html)
```

Reply:
```suggestion
* [Main band and sub-footer band only (default implementation)](https://wet-boew.github.io/GCWeb/sites/footers/no-footer-contextual-en.html)
```
I would change it to this to showcase the default state as an alternate version of the complete footer since it's under the heading 'Alternate implementation'